.github/workflows/spec.yml

```diff
@@ -1,4 +1,4 @@
-name: Spec
+name: Dart CI
```

Comment:
Accidental change?

Reply:
Oh I see how this actually shows up, seems fine to keep the new name

Reply:
No this is just to make the PR checks line up a bit better visually:

<img width="377" alt="Screenshot 2023-09-15 at 11 17 57 AM" src="https://github.com/dart-lang/language/assets/1269969/b2a33a16-a315-4b2a-a6f4-addd3135a915">